test/session_test.exs

```diff
@@ -39,6 +38,7 @@ defmodule Janus.SessionTest do
       assert_receive {:trace, ^conn, :receive, %{"janus" => "ack"}}, 2 * interval
     end
 
+    @moduletag :capture_log
```

Comment:
If you really intended to put `@moduletag` add it on top of the module. Usually `@tag` is placed here